src/framework/preferences/js/FullPreviewPrefsEditor.js

```diff
@@ -41,7 +41,9 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
                         }
                     },
                     listeners: {
-                        "onReady.boilOnPrefsEditorReady": "{fullPreview}.events.onPrefsEditorReady"
+                        "onReady.boil": {
```

Comment:
Similarly this looks like stray material from another pull

Reply:
This is so that it will properly override the "onReady.boil" listener from the prefsEditorLoader base grade. https://github.com/jobara/infusion/blob/6c4381917a85641beb6195a50006976f7ebad733/src/framework/preferences/js/PrefsEditor.js#L47

Reply:
OK, I see